next.config.js

```diff
@@ -1,48 +1,57 @@
-// This guard clause allows the app to still build in the event another exception handler will be used,
-// or the sentry project hasn't been set up yet
-if (!process.env.SENTRY_DSN) return
-
 // This file sets a custom webpack configuration to use your Next.js app
 // with Sentry.
 // https://nextjs.org/docs/api-reference/next.config.js/introduction
 // https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/
 const { withSentryConfig } = require('@sentry/nextjs')
+const SENTRY_DSN = process.env.SENTRY_DSN
 
-const nextConfig = {
+let nextConfig = {
   output: 'standalone',
   reactStrictMode: true,
   swcMinify: true,
   eslint: {
-    dirs: ['pages', 'utils'], // Only run ESLint on the 'pages' and 'utils' directories during production builds
-  },
-  sentry: {
-    // Use `hidden-source-map` rather than `source-map` as the Webpack `devtool`
-    // for client-side builds. (This will be the default starting in
-    // `@sentry/nextjs` version 8.0.0.) See
-    // https://webpack.js.org/configuration/devtool/ and
-    // https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#use-hidden-source-map
-    // for more information.
-    hideSourceMaps: true,
+    // Warning: This allows production builds to successfully complete even if
+    // your project has ESLint errors.
+    // ref: https://nextjs.org/docs/pages/api-reference/next-config-js/eslint
+    ignoreDuringBuilds: true,
```

Comment:
I confirmed that this works by pushing this branch to production on vercel.
<img width="1285" alt="Screenshot 2023-12-15 at 6 35 25 PM" src="https://github.com/scientist-softserv/webstore/assets/29032869/ebd5d476-95dd-446f-a854-67096c626cca">